DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Dashboard/RankingController.cs

```diff
@@ -12,20 +14,46 @@
     [Route("/TrackingSystem/Centre/Ranking")]
     public class RankingController : Controller
     {
+        private static readonly (int, string) Week = (7, "Week");
+        private static readonly (int, string) Fortnight = (14, "Fortnight");
+        private static readonly (int, string) Month = (30, "Month");
+        private static readonly (int, string) Quarter = (91, "Quarter");
+        private static readonly (int, string) Year = (365, "Year");

```

Comment:
Should this be in a Period enum class? We could have id, name, and numberOfDays properties per field, and I think there should be a .GetAll() method on the base Enumeration class.